framework-plugins/lisk-framework-chain-connector-plugin/test/unit/plugin.spec.ts

```diff
@@ -84,6 +147,33 @@ describe('ChainConnectorPlugin', () => {
 		subscribe: jest.fn(),
 	};
 
+	const setupChainConnectorPluginLoad = async (block: Block, eventsCount = 2) => {
```

Comment:
maybe this function can take mocks as an argument as well? also, since I'm working on updating the flow that will require a lot of refactoring of functions, lets keep it simple and avoid refactoring other parts not related to the issue